index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Static HTML quiz</title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+	<article class="productCard">
+		<header class="productTitle">
+			<h1>Dunk Cards</h1>
+		</header>
+		<section class="productInfo">
+			<img src="dunkcard" alt="yellow dunk card" />
+			<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Nobis id laborum harum molestiae, illum cupiditate. Consequatur distinctio aspernatur, fuga odio non sapiente, id repudiandae necessitatibus saepe odit dolor possimus doloremque.</p>
```

Comment:
Requirement 5 calls for 3 block elements, but <img> is an inline element. Either change it to a block element or wrap it in a block element, then make sure it is still centered.

On a side note, next time crop the image closer ;P